articles/quickstart/native/react-native/00-login.md

```diff
@@ -57,47 +57,20 @@ The first step in adding authentication to your application is to provide a way
 
 ### Configure Android
 
-In the file `android/app/src/main/AndroidManifest.xml` you must make sure the activity you are going to receive the authentication on has a **launchMode** value of `singleTask` and that it declares the following intent filter (see the [React Native docs](https://facebook.github.io/react-native/docs/linking#basic-usage) for more information):
-
-```xml
-<intent-filter>
-    <action android:name="android.intent.action.VIEW" />
-    <category android:name="android.intent.category.DEFAULT" />
-    <category android:name="android.intent.category.BROWSABLE" />
-    <data
-        android:host="${account.namespace}"
-        android:pathPrefix="/android/<%= "${applicationId}" %>/callback"
-        android:scheme="<%= "${applicationId}" %>" />
-</intent-filter>
-```
-
-The sample app declares this inside the **MainActivity** like this:
-
-```xml
-<activity
-android:name=".MainActivity"
-android:label="@string/app_name"
-android:launchMode="singleTask"
-android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
-android:windowSoftInputMode="adjustResize">
-<intent-filter>
-    <action android:name="android.intent.action.MAIN" />
-    <category android:name="android.intent.category.LAUNCHER" />
-</intent-filter>
-<intent-filter>
-    <action android:name="android.intent.action.VIEW" />
-    <category android:name="android.intent.category.DEFAULT" />
-    <category android:name="android.intent.category.BROWSABLE" />
-    <data
-        android:host="${account.namespace}"
-        android:pathPrefix="/android/<%= "${applicationId}" %>/callback"
-        android:scheme="<%= "${applicationId}" %>" />
-</intent-filter>
-</activity>
+Open your app's `build.gradle` file (typically at `android/app/build.gradle`) and add the following manifest placeholders:
+
+```groovy
+android {
+    defaultConfig {
+        // Add the next line
+        manifestPlaceholders = [auth0Domain: "${account.namespace}", auth0Scheme: "${applicationId}"]
```

Comment:
I think these need to be escaped for the build to complete successfully. See the previous line for reference https://github.com/auth0/docs/pull/9768/files#diff-0a0acd30f658f139c1feac53f542219f8c9441c8de17fd5d27981439ae022d09L93.